public/javascripts/script.js

```diff
@@ -1,34 +1,35 @@
 $(document).ready( function ( ) {
-  $('.delete-book').on('click', () => {
-    const id = $(this).data.id
-    console.log("id from script",id)
-    let url = '/delete/'+ id
-    if(confirm('Delete Book?')) {
-      $.ajax({
-        url: url,
-        type: 'POST',
-        success: function(result){
-          console.log('Deleting Book...')
-          window.location.href='/'
-        },
-        error: function(err){
-          console.log(err)
-        }
-      })
-    }
-  })
 
-  $('.edit-book').on('click', (event) => {
-    const id = $(event.currentTarget).data('id')
+  // $('.delete-book').on('click', (event) => {
```

Comment:
We may use it to create some front-end sexiness... carousels or something like that lol